python/whylogs/extras/image_metric.py

```diff
@@ -101,11 +102,32 @@ def image_based_metadata(img):
     }
 
 
+@dataclass(frozen=True)
+class ImageMetricConfig(MetricConfig):
+    allowed_exif_tags: Set[str] = field(default_factory=set)
+    forbidden_exif_tags: Set[str] = field(default_factory=set)
+
+
+def _trackable_exif_tag(exif_tag: str, allowed_tags: Set[str], forbidden_tags: Set[str]) -> bool:
```

Comment:
Doesn't this need to be tabbed in to be under the class? the ones before are tabbed in so I assume this was supposed to be apart of the class not on its own?

Reply:
Stylistically I liked this better than `self._trackable_exif_tag(tag_name)`. If it starts growing a lot more arguments that are members of the `ImageMetric` I'd probably change my mind.

Reply:
I might be persuaded that `self._wants_to_track(tag_name)` reads nicely...